### ISSUE TITLE
implemented action to refresh annotation(s) of node(s) or map(s)

### ISSUE THREAD

**Issue by stlanger** [Docear/Desktop#136]
we now have the option to right-click a PDF node in a mind map and select to import all new annotations. However, there is no action to refresh the annotations (compare them to the ones in the PDF files, update them or show any conflictions)

**Comment by DonCN** [comment]
Yes, If we select "Import all annotations", all the notes show again, there will be two copy of notes.
